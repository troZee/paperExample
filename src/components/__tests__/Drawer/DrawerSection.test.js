import React from 'react';
import { View } from 'react-native';
import renderer from 'react-test-renderer';
import DrawerSection from '../../Drawer/DrawerSection.tsx';

describe('DrawerSection', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(
        <DrawerSection>
          <View />
        </DrawerSection>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
