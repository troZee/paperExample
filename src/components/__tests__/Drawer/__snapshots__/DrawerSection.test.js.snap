// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrawerSection renders properly 1`] = `
<View
  style={
    Array [
      Object {
        "marginBottom": 4,
      },
      undefined,
    ]
  }
>
  <View />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "rgba(0, 0, 0, 0.12)",
          "height": 0.5,
        },
        undefined,
        Object {
          "marginTop": 4,
        },
      ]
    }
  />
</View>
`;
